tests/test.bats

```diff
@@ -104,3 +104,13 @@ function test_datastore_request () {
     return 1;
   fi
 }
+
+@test "datapusher working" {
+  datapusher_status=$(curl -X GET "http://datapusher:8000/status" | grep -o "push_to_datastore")
```

Comment:
Please add the `--fail` flag so a non 200 status code results in curl failing. Also, no need to `grep` and capture the output, `grep` will return non-zero and fail the test if it doesn't find the pattern. No need for the `if`/`else`. The test is simply:

```suggestion
  curl --silent --fail "http://datapusher:8000/status" | grep push_to_datastore
```